### ISSUE TITLE
v3 Editor slow to initialize compared to v2

### ISSUE THREAD

**Issue by dvriezen** [froala/react-froala-wysiwyg#206]
I just started updating to 3.0.6 from 2.9.1 and everything is fairly straight forward except v3 seems to have a considerably longer initialization than v2 had. It works fine once loaded but takes about a half-second longer on initialization than it used to. This causes the screen to flash as the editor has a height of 0 for a split second, despite having a heightMin set in config.

Did something change around how this should be configured or loaded to avoid this delay or is this just inevitable with the new implementation in v3?

**Comment by iernie** [comment]
Is there a fix for this?

**Comment by dvriezen** [comment]
@iernie I ended up completely rebuilding the implementation I was using into a hook component which seemed to help. There is still a flash that I was never able to fix but it occurs quickly within the first few milliseconds for me rather than after a second or two.

I closed this as I also wasn't sure how much of the issue was caused by the Codox-Froala integration I was implementing at the time. I'm still not sure how much that played into this issue.
